bookdown/01-basics/01-binary.Rmd

```diff
@@ -35,25 +36,25 @@ The element $C_{21}$ is called False Negatives (FN).
 
 We can now normalize in rows and columns of the confusion matrix to derive several informative metrics:
 
-- **True Positive Rate (TPR)**: How many of the true positives did we predict as positive?
-- **True Negative Rate (TNR)**: How many of the true negatives did we predict as negative?
-- **Positive Predictive Value PPV**: If we predict positive how likely is it a true positive?
-- **Negative Predictive Value NPV**: If we predict negative how likely is it a true negative?
+* **True Positive Rate (TPR)**: How many of the true positives did we predict as positive?
+* **True Negative Rate (TNR)**: How many of the true negatives did we predict as negative?
+* **Positive Predictive Value PPV**: If we predict positive how likely is it a true positive?
+* **Negative Predictive Value NPV**: If we predict negative how likely is it a true negative?
 
 ```{r 01-binary-003, echo = FALSE}
 knitr::include_graphics("images/confusion_matrix(wikipedia).png")
 ```
+
 Source: [Wikipedia](https://en.wikipedia.org/wiki/Evaluation_of_binary_classifiers)
 
 It is difficult to achieve a high TPR and low FPR in conjunction, so one uses them for constructing the ROC Curve.
 We characterize a classifier by its TPR and FPR values and plot them in a coordinate system.
-The best classifier lies on the top-left corner, while the worst classifier lies at the diogonal.
-<!-- FIXME Why is the best classifier on the top-left? The argument here only shows that classifiers at the diogonal are inferior but there is no argument in place, illustrating why the other (top left) classifiers are superior. -->
```

Comment:
yeah I added the FIXME back then but I was not sure if it was something that has to be explained or perhaps people know already that are familiar with mlr. But yeah I am not objecting towards keeping it if it may be something not that self explanatory.

Reply:
I guess Bernd put it there (idk). Let' just leave it there for now until its really fixed.

Reply:
good point, lets keep it.